agent/containers/images/Makefile

```diff
@@ -198,8 +198,8 @@ centos-7-base.Dockerfile: Dockerfile.base.j2 epel-7-pbench.repo
 fedora-33-base.Dockerfile: Dockerfile.base.j2 fedora-33-pbench.repo
 	jinja2 Dockerfile.base.j2 -D pbench_repo_file=fedora-33-pbench.repo -D pkgmgr=dnf -D distro_image=fedora:33 -D distro_image_name="Fedora 33" -o $@
 
-fedora-32-base.Dockerfile: Dockerfile.base.j2 fedora-32-pbench.repo
-	jinja2 Dockerfile.base.j2 -D pbench_repo_file=fedora-32-pbench.repo -D pkgmgr=dnf -D distro_image=fedora:32 -D distro_image_name="Fedora 32" -o $@
+fedora-34-base.Dockerfile: Dockerfile.base.j2 fedora-34-pbench.repo
+	jinja2 Dockerfile.base.j2 -D pbench_repo_file=fedora-34-pbench.repo -D pkgmgr=dnf -D distro_image=fedora:34 -D distro_image_name="Fedora 34" -o $@
```

Comment:
Just a nit, but this changes the order of the Fedora targets from decreasing to increasing (while the Centos targets are still in decreasing order).

Reply:
Was trying to reduce the amount of differences.

Reply:
Understood, but, overall there are a small number of changes, and future readers will most likely be looking at the whole file rather than just the diffs.  And, in this case, the diffs appear (more or less) to be "this _line_ removed and this _line_ inserted", which would still be the case if you positioned them properly (and, it's not hard to see what the differences are within these two lines).

I almost wonder whether there shouldn't be a single "fedora" target using a pattern target form with the `33` or `34` just plugged in where it goes.  E.g., I think something like this would "just work":
```
fedora-%-base.Dockerfile: Dockerfile.base.j2 fedora-%-pbench.repo
	jinja2 Dockerfile.base.j2 -D pbench_repo_file=fedora-$*-pbench.repo -D pkgmgr=dnf -D distro_image=fedora:$* -D distro_image_name="Fedora $*" -o $@
```
although the value for `distro_image_name` might need some additional handling.

Reply:
Tracked in #2380.